index2.html

```diff
@@ -36,8 +36,244 @@
+      </div>
+    </section>
+    <button type="submit" class="resume-btn">See Project</button>
+
+    <div class="language-container">
+      <div class="language-column">
+        <h1 class="languages2">Languages</h1>
+        <img class="globe" alt="globe picture" src="ball.svg" />
+      </div>
+    </div>
+    <div class="center-row">
+      <ul>
+        <li class="java-language">Javascript</li>
+        <li class="ruby-language">Ruby</li>
+        <li class="html-language">HTML</li>
+        <li class="css-language">CSS</li>
+      </ul>
+    </div>
+
+    <div class="language-container">
+      <div class="language-column">
+        <h1 class="frameworks2">FrameWorks</h1>
+        <img class="ficon" alt="ficon" src="frame.svg" />
+      </div>
+      <div class="top-row">
+        <ul>
+          <li class="boot">Bootstrap</li>
+          <li class="ruby-rails">Ruby on rails</li>
+          <li class="r-spec">RSpec</li>
+        </ul>
+      </div>
+
+      <div class="bottom-row">
+        <ul>
+          <li class="Capybara">Capybara</li>
+          <li class="Sele">Selenium</li>
+        </ul>
+      </div>
+    </div>
+    <div class="language-container">
+      <div class="language-column">
+        <h1 class="skills">Skills</h1>
+        <img class="light" alt="" src="lighting.svg" />
+      </div>
+      <div class="top-row">
+        <ul>
+          <li class="codekick">Codekick</li>
+          <li class="github">Github</li>
+          <li class="codepen">Codepen</li>
+          <li class="gitlab">Gitlab</li>
+        </ul>
+      </div>
+      <div class="bottom-row">
+        <ul>
+          <li class="term">Terminal</li>
+        </ul>
+      </div>
+    </div>
+
+    <section class="projects">
+      I'm always interested in hearing about new projects, so if you'd like to
+      chat please get in touch.
+    </section>
+
+    <footer class="contactpage">
+      <form action="https://formspree.io/f/myyvzkag" method="post">
+        <ul>
+          <li>
+            <input
+              id="Name"
+              name="user_name"
+              type="text"
+              maxlength="30"
+              class="name"
+              placeholder="Full Name"
+              required="required"
+            />
+          </li>
+
+          <li>
+            <input
+              id="email"
+              name="user_email"
+              type="email"
+              class="name-last"
+              placeholder="Email Address"
+              required="required"
+            />
+          </li>
+          <li>
+            <textarea
+              id="text-box"
+              name="message"
+              maxlength="500"
+              class="enter"
+              placeholder="Enter text here"
+              required="required"
+            >
+            </textarea>
+          </li>
+        </ul>
+
+        <button type="submit" class="touch-btn">Get in touch</button>
+      </form>
```

Comment:
- [ ] Kindly adjust your padding, margin, and typography on the form `input` and `textarea`, in order to make our form looks exactly like the provided Figma design. :wink: 

   - ### Your design

   ![image](https://user-images.githubusercontent.com/87197412/186275583-7e33a959-0c66-403f-966f-d6d9fc27bb09.png)

   - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/186275657-72ab8b34-db3d-41d0-976d-9e9dd1fa06f5.png)